llm_api_hub/server/app.py

```
@@ -34,6 +34,16 @@
 
 REPO_ROOT = Path(__file__).resolve().parents[2]
 MULTI_AI_CLI = REPO_ROOT / "multi-ai-cli"
+
+# Ensure multi-ai-cli is in path for core imports
+if str(MULTI_AI_CLI) not in sys.path:
+    sys.path.insert(0, str(MULTI_AI_CLI))
+
+try:
+    from core import provider_checklist, provider_registry
```

Pylint (4.0.6)

[warning] Unused provider_registry imported from core

(W0611)